api/src/main/java/net/luckperms/api/model/PermissionHolder.java

```diff
@@ -299,6 +299,26 @@
      */
     @NonNull @Unmodifiable Collection<Group> getInheritedGroups(@NonNull QueryOptions queryOptions);
 
+    /**
+     * Gets a collection of the {@link Group}s this holder inherits nodes from.
+     *
+     * <p>This method exists to avoid the verbosity of converting an existing {@link QueryOptions}
+     * to its builder form and back solely to change the {@link Flag#RESOLVE_INHERITANCE} flag.</p>
+     *
+     * <p>This calls {@link #getInheritedGroups(QueryOptions)} with the modified query options.
+     * See the description for it for the specific behavior of this method.</p>
+     *
+     * @param queryOptions       the query options
+     * @param resolveInheritance whether the resulting collection should include inherited groups
+     *                           beyond the holder's direct parent groups
+     * @return a collection of the groups the holder inherits from
+     * @since 5.4
+     * @see #getInheritedGroups(QueryOptions)
+     */
+    default @NonNull @Unmodifiable Collection<Group> getInheritedGroups(@NonNull QueryOptions queryOptions, boolean resolveInheritance) {
```

Comment:
`getInheritedGroups` ... without inheritance? That doesn't seem right.

Reply:
Yeah I guess it can be `getParentGroups` only with no extra parameters then; I thought of adding this method initially because I often needed to get the direct parent groups of a user, it does add up and it does yield some ugly looking code.